.github/workflows/stockfish.yml

```diff
@@ -251,6 +253,17 @@ jobs:
           make clean
           make -j2 ARCH=x86-64-vnni256 build
 
+      # armv8 tests
+
+      - name: Test armv8 build
+        if: ${{ matrix.config.run_armv8_tests }}
+        run: |
+          export PATH=$ANDROID_NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/bin:$PATH
+          export LDFLAGS="-static -Wno-unused-command-line-argument"
+          make clean
+          make -j2 ARCH=armv8 build
+          ../tests/signature.sh $benchref
```

Comment:
since we just built an ARM binary, and I assume this is a cross, does the signature.sh script automatically run this under qemu ? 

If so, can't we specify the PGOBENCH command to even do a profile-build make?

Reply:
PGO for armv8 in test
https://github.com/ppigazzini/Stockfish/runs/5096366759?check_suite_focus=true

Reply:
PGO armv8 failed :(
```
Step 2/4. Running benchmark for pgo-build ...
./stockfish bench 2>&1 | tail -n 4
Segmentation fault (core dumped)
Total time (ms) : 106686
Nodes searched  : 4684080
Nodes/second    : 43905
qemu: uncaught target signal 11 (Segmentation fault) - core dumped
```